README.md

```diff
@@ -33,10 +33,21 @@ This will be akin to the Web API that you built in the last sprint, only this ti
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 1. Explain the difference between `RDBMS` and `SQL`.
+
+- Relational database management system is the basis for SQL
+
 1. Why do tables need a `primary key`?
+
+- tables need a primary key to differentiate rows from each other
+
 1. What is the name given to a table column that references the primary key on another table.
+
+- Foreign key
+
 1. What do we need in order to have a _many to many_ relationship between two tables.
 
+- A new table with two foreign keys connecting the two tables
```

Comment:
and, to reduce redundancies, the best practice is to have both of those foreign keys be the (composite) primary key.